config/rbac/role.yaml

```diff
@@ -12,6 +12,25 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - get
+  - list
+  - patch
+  - update
```

Comment:
interesting ... for my education, why does the controller need to update serviceaccounts? .... binding secrets to the SAs?

Reply:
sounds good thanks